### ISSUE TITLE
When running install: "ERROR: Cannot dynamically create an instance of type 'System.Security.Cryptography.MD5CryptoServiceProvider'"

### ISSUE THREAD

**Issue by taurawaxeye** [sttz/install-unity#29]
when running install from the current directory

```
./install-unity install 2018.2 --packages Unity,Documentation
```

I get the following error

```
ERROR: Cannot dynamically create an instance of type 'System.Security.Cryptography.MD5CryptoServiceProvider'. Reason: No parameterless constructor defined. (retrying in 5s, 3 retries remaining)·························║       0 B of 944.74 MB @       0 B/sERROR: Cannot dynamically create an instance of type 'System.Security.Cryptography.MD5CryptoServiceProvider'. Reason: No parametERROR: Cannot dynamically create an instance of type 'System.Security.Cryptography.MD5CryptoServiceProvider'. Reason: No parameterless constructor defined. (retrying in 5s, 2 retries remaining)·························║       0 B of 944.74 MB @       0 B/sERROR: Cannot dynamically create an instance of type 'System.Security.Cryptography.MD5CryptoServiceProvider'. Reason: No parametERROR: Cannot dynamically create an instance of type 'System.Security.Cryptography.MD5CryptoServiceProvider'. Reason: No parameterless constructor defined. (retrying in 5s, 1 retries remaining)·························║       0 B of 944.74 MB @       0 B/sERROR: Cannot dynamically create an instance of type 'System.Security.Cryptography.MD5CryptoServiceProvider'. Reason:ERROR: Cannot dynamically create an instance of type 'System.Security.Cryptography.MD5CryptoServiceProvider'. Reason: No parameterless constructor defined. (retrying in 5s, 0 retries remaining)·······························································································║       0 B of 944.74 MB @       0 B/sERROR: Cannot dynamically create an instance of type 'System.Security.Cryptography.MDWARN:  Progress bar disabled due to exception during rendering: System.AggregateException: One or more errors occurred. (One or more errors occurred. (Cannot dynamically create an instance of type 'System.Security.Cryptography.MD5CryptoServiceProvider'. Reason: No parameterless constructor defined.))                                                                                        ---> System.AggregateException: One or more errors occurred. (Cannot dynamically create an instance of type 'System.Security.Cryptography.MD5CryptoServiceProvider'. Reason: No parameterless constructor defined.)                                                                                                                                                                                 ---> System.MissingMethodException: Cannot dynamically create an instance of type 'System.Security.Cryptography.MD5CryptoServiceProvider'. Reason: No parameterless constructor defined.         
   at System.RuntimeType.ActivatorCache..ctor(RuntimeType )
   at System.RuntimeType.CreateInstanceDefaultCtor(Boolean , Boolean )
   at System.Activator.CreateInstance(Type type, Boolean nonPublic, Boolean wrapExceptions)
   at System.Activator.CreateInstance(Type type)
   at sttz.InstallUnity.Downloader.Start(CancellationToken cancellation)
   --- End of inner exception stack trace ---
   at sttz.InstallUnity.UnityInstaller.Process(InstallStep steps, Queue queue, Boolean skipChecks, CancellationToken cancellation)
   at sttz.InstallUnity.UnityInstaller.Process(InstallStep steps, Queue queue, Boolean skipChecks, CancellationToken cancellation)
   --- End of inner exception stack trace ---
   at sttz.InstallUnity.InstallUnityCLI.Install()
One or more errors occurred. (Cannot dynamically create an instance of type 'System.Security.Cryptography.MD5CryptoServiceProvider'. Reason: No parameterless constructor defined.)
Cannot dynamically create an instance of type 'System.Security.Cryptography.MD5CryptoServiceProvider'. Reason: No parameterless constructor defined.
```

is the machine I am running on missing any dependencies ?
as this program is the only way on the internet I can find that is able to the download unity via command line.

**Comment by sttz** [comment]
Oh, thanks, I messed up when changing from a NativeAOT to a .Net Single File build.

Should be fixed with install-unity 2.10.1.

**Comment by taurawaxeye** [comment]
I can confirm the latest version now works. 

Thanks.
